src/components/comment-input.tsx

```diff
@@ -86,7 +84,10 @@ export const CommentInput: React.VoidFunctionComponent = () => {
 
   return (
     <InputContainer>
-      <AvatarImage src={user.avatarUrl} alt={user.name} />
+      <AvatarImage
+        src={generateAvatarUrl(user.picture, user.email)}
```

Comment:
Since we now use `useUser` as is, we have access to our own email, but our picture does not include the gravatar fallback. So we supplement the value with a fallback if needed. We can move this logic into `<AvatarImage />` later to reduce repetition.